packages/shared/src/components/notifications/NotificationItem.tsx

```diff
@@ -51,47 +52,45 @@ function NotificationItem({
       .filter((avatar) => avatar) ?? [];
   const hasAvatar = avatarComponents.length > 0;
 
-  const onClick = () => {
-    router.push(targetUrl);
-  };
-
   return (
-    <button
-      type="button"
-      onClick={onClick}
-      className={classNames(
-        'flex flex-row py-4 pl-6 pr-4 hover:bg-theme-hover focus:bg-theme-active',
-        isUnread && 'bg-theme-float',
-      )}
-    >
-      <NotificationItemIcon icon={icon} />
-      <div className="flex flex-col flex-1 ml-4 w-full text-left typo-callout">
-        {hasAvatar && (
-          <span className="flex flex-row gap-2 mb-4">{avatarComponents}</span>
+    <Link href={targetUrl} passHref>
+      <a
```

Comment:
Please use `button` if it doesnt have a target URL, and use `a` if it does with the href

Reply:
Every notification will have a `targetUrl`.